cudapoa/samples/sample_cudapoa.cpp

```diff
@@ -227,7 +246,7 @@ int main(int argc, char** argv)
 
     if (long_read)
     {
-        generate_simulated_long_reads(windows, batch_size);
+        generate_bonito_long_reads(windows, batch_size);
     }
```

Comment:
I was not sure whether to keep or remove `generate_simulated_long_reads`, since now it's been replaced with `generate_bonito_long_reads`, there's no use for it other than debugging and internal benchmarking.